board/OrangePi-Plus2E/overlay/boot/loader.rc.local

```diff
@@ -0,0 +1,2 @@
+load kernel
+load -t dtb boot/dtb/orangepi-plus-2e.dtb
```

Comment:
This shouldn't be needed.
ubldr load the dtb after the uboot variable "fdtfile"